spec/domain/queries/find_content_spec.rb

```diff
@@ -348,5 +347,57 @@
         hash_including(base_path: '/this/is/a/big/base-path'),
       )
     end
+
+    describe 'search by base_path with full URLs' do
+      before do
+        edition1 = create :edition, base_path: '/base-path', date: 2.months.ago, organisation_id: primary_org_id
+
+        create :metric, edition: edition1, date: 15.days.ago
+        recalculate_aggregations!
+      end
+
+      let(:results) { described_class.call(filter: filter.merge(search_term: search_term))[:results] }
```

Comment:
I think this should be a `subject` if you use it directly in your expectations

Reply:
Would it be simpler to unit test the parse_search_term function?

Reply:
I would rather keeping `parse_search_term` as `private`. To test it I would need to make it public.